src/super_gradients/recipes/coco2017_yolox.yaml

```diff
@@ -0,0 +1,46 @@
+# YoloX Detection training on CoCo2017 Dataset:
+# YoloX trained in 640x640 mAP@0.5-0.95 (COCO API, confidence 0.01, IoU threshold 0.65, test on 640x640 images)
+# Checkpoints + tensorboards: https://deci-pretrained-models.s3.amazonaws.com/yolox_coco/
+# Recipe runs with batch size = 16 X 8 gpus = 128.
+# Instructions:
+# python -m torch.distributed.launch --nproc_per_node=8 --config-name=coco2017_yolox architecture=yolox_<YOLOX_SIZE> dataset_params.data_dir=<YOUR_COCO_LOCAL_PATH> ckpt_root_dir=<CHEKPOINT_DIRECTORY>
+# Training times and accuracies:
+# yolox_n: 1d 16h 33m 9s on 8 NVIDIA GeForce RTX 3090, mAP: 26.26
+# yolox_tiny: 20h 43m 37s on 8 NVIDIA RTX A5000, mAP: 37.0
+# yolox_s: 1d 17h 40m 30s on 8 NVIDIA RTX A5000, mAP: 40.46
+# yolox_m: 1d 22h 23m 43s on 8 NVIDIA GeForce RTX 3090, mAP: 47.18
+# yolox_l: 2d 14h 11m 41s on 8 NVIDIA GeForce RTX 3090, mAP: 52.25
+
+defaults:
+  - training_hyperparams: coco2017_yolox_train_params
+  - dataset_params: coco_detection_yolox_dataset_params
+  - arch_params: yolov5s_arch_params
```

Comment:
Now that you have all yolox yamls, put it here instead of yolov5s_arch_params

Reply:
I think this is when anchors anchors from the yaml will cause DDP to crash

Reply:
We currently just override the anchors...Ill remove them completely soon, but not in this PR.

Reply:
Anyway, arch_params here shouldn't be for yolov5s. If it can be changed now then great

Reply:
used yolox_s arch params in  https://github.com/Deci-AI/super-gradients/commit/733237b8f40c7d6fbb911edc67d36201d0c131af